### PR TITLE
Show user pubkey in settings

### LINKED ISSUE #281: Show pubkey of users logged in with Lightning
A user asked how they can check which wallet they used for login: https://stacker.news/items/179376

We could show users their pubkey in their profiles so they can compare with the pubkey in their wallet.

I know that at least Phoenix shows your node id (pubkey) in the channel list.

### PR DESCRIPTION
**Pull request by ekzyis**
Closes #281 

Looks like this:

2023-05-17-154052_1920x1080_scrot

(shows full pubkey when hovering)

I think it could look better but don't know how exactly :thinking:

### CODE REVIEW

File: api/typeDefs/user.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default gql`
   }
 
   type AuthMethods {
-    lightning: Boolean!
+    lightning: String!
     slashtags: Boolean!
```
Comment (ekzyis) [inline api/typeDefs/user.js:37]: Maybe do the same with slashtags?

**Comment by ekzyis** [comment]
> This pubkey is specific to SN given the way ln-auth works. It is not their node pubkey.

Oh, right, it creates domain-specific `linkingKey`s... I forgot

**Comment by ekzyis** [comment]
> which I was going to do when I allowed multiple ln auths on a single account

Makes sense. I'll link #164 and close this PR then. Sorry for the mistake, I realize I am getting to eager to change stuff without properly thinking first hehe. Maybe related to "too much excitement" haha

**Comment by huumn** [comment]
All good. I strongly prefer you eagerly changing stuff than not. Especially if you don't mind it occasionally not getting through when it isn't right.

**Pull request closed without merging**